packages/maskbook/src/components/InjectedComponents/TypedMessageRenderer.tsx

```diff
@@ -29,6 +29,7 @@ export interface TypedMessageRendererProps<T extends TypedMessage> {
      * The TypedMessage
      */
     message: T
+    image?: string | Blob
```

Comment:
this is not correct. `TypedMessageRendererProps` is the very base case. All info you need should be in the `message` field